src/scenarios/VariousInputs.scenario.tsx

```diff
@@ -60,8 +63,24 @@ const InputField: React.FC<InputFieldProps> = ({
                 onChange={
                   onChange as React.ChangeEventHandler<HTMLInputElement>
                 }
-                sx={{ width: '100%' }}
+                sx={{ width: '80%' }}
                 inputProps={{ 'data-testid': `${type}-input` }}
+                InputProps={{
```

Comment:
This is intentional with `inputProps` and `InputProps` - see https://stackoverflow.com/a/69872110 for a more detailed answer.